data_hub_api/docmaps/v2/api_input_typing.py

```diff
@@ -69,6 +69,9 @@
         'collection_curator_name': Optional[str],
         'is_collection_curator_et_al': Optional[bool],
         'collection_thumbnail_url': Optional[str],
+        'podcast_id': Optional[str],
```

Comment:
It might be beneficial to define separate types for collection and podcast. I am a bit suspicious that all of the fields are optional?

Reply:
Yes, podcast id should be integer. I will fix it. But all of the fields are optional as not every manuscript has related podcast, collection or article.

Reply:
I think I made the comment not yet understanding how it is put together. Somehow I thought there would be a list of items that is one of the those types (related article, collection or podcast). But I realise now that we populate the fields of the different types on the same item.